applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp

```diff
@@ -197,6 +197,32 @@ class MechanicalExplicitStrategy
         return mReformDofSetAtEachStep;
     }
 
+    /**
+     * @brief This function is designed to move the mesh
+     * @note Be careful it just consider displacements, derive this method to adapt to your own strategies (ALE, FSI, etc...)
+     */
+    void MoveMesh() override
```

Comment:
blocking bcs of this

Reply:
@philbucher in explicit calculations time is critical and the "MESH MOVED" is accessing to the buffer, which is absurdly slow

Reply:
Can you propose a better solution?

Reply:
I did:
> => use KRATOS_WARNING as echo-level, then it will not print it (until a better solution in the Analysis is found)